### PR TITLE
Reload

### PR DESCRIPTION
**Pull request by tomer-epstein**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -437,16 +438,24 @@ export default {
     },
     toggleConsole() {
       this.showConsole = !this.showConsole;
+    },
+    init() {
+      this.yeomanName = "<no generator selected>";
+      this.promptIndex = 0;
```
Comment (slavik-lvovsky) [inline frontend/src/App.vue]: this one is already defined in data. No need to do it again

File: frontend/src/App.vue [hunk 2]
```diff
@@ -437,16 +438,24 @@ export default {
...
+      this.prompts = [];
```
Comment (slavik-lvovsky) [inline frontend/src/App.vue]: this one is already defined in data. No need to do it again

File: frontend/jest.config.js [hunk 3]
```diff
@@ -36,9 +36,9 @@ module.exports = {
   coverageThreshold: {
     "global": {
       "branches": 91.7,
-      "functions": 98.5,
-      "lines": 96.3,
-      "statements": 96.3
+      "functions": 97.14,
+      "lines": 95.97,
+      "statements": 95.98
```
Comment (slavik-lvovsky) [inline frontend/jest.config.js]: you need to add few more tests.
Just run the npm test and check in coverage/index.html what lines are still not covered

Comment (slavik-lvovsky) [reply frontend/jest.config.js]: I saw your tests and they look ok. They should cover all new lines.
Why do you need to decrease coverage ?

File: frontend/src/App.vue (hunk 1 above)
Comment (tomer-epstein) [reply frontend/src/App.vue]: Must-to do it agian. here. next time data() is called ia after invoking 'receiveIsWebviewReady'

File: frontend/src/App.vue (hunk 2 above)
Comment (tomer-epstein) [reply frontend/src/App.vue]: Must-to do it agian. here. next time data() is called ia after invoking 'receiveIsWebviewReady'

Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Ok, i understand. No problem

**Comment by slavik-lvovsky** [comment]
Your tests looks ok, i give you approve, just please check why your coverage is lower than it was. It is very strange.

**Pull request merged**